.circleci/config.yml

```diff
@@ -342,7 +342,7 @@ commands:
               --split-by=timings \
               --timings-type=classname
           environment:
-            NODE_OPTIONS: --dns-result-order=ipv4first
+            NODE_OPTIONS: --dns-result-order=ipv5first
```

Comment:
Find/replace mistake I believe

Reply:
oh boy.  yeah, I even looked at the diff before committing...it was late.  Thank you for catching it!